Formula/metricbeat.rb

```diff
@@ -72,16 +79,16 @@ def plist; <<~EOS
     (testpath/"logs").mkpath
     (testpath/"data").mkpath
 
-    metricbeat_pid = fork do
-      exec bin/"metricbeat", "-c", testpath/"metricbeat.yml",
-      "--path.data=#{testpath}/data", "--path.logs=#{testpath}/logs"
+    pid = fork do
+      exec bin/"metricbeat", "-path.config", testpath/"config", "-path.data", testpath/"data"
     end
 
     begin
-      sleep 2
+      sleep 30
```

Comment:
why so long?

Reply:
Because the changes don’t get flushed to disk fast enough so the test ends up being flaky with a short sleep.